Fails to build on Debian testing

When I try to build this on my Debian testing system, I get a build error:
```
mkdir build
cd build
cmake ..
make -j8
```
shows
```
uru/libhsplasma/core/PRP/Animation/plWindBoneMod.cpp: In member function ‘virtual void plWindBoneMod::removeTarget(const plKey&)’:
uru/libhsplasma/core/PRP/Animation/plWindBoneMod.cpp:107:65: error: no matching function for call to ‘find(std::vector<plKey>::iterator, std::vector<plKey>::iterator, const plKey&)’
  107 |     auto it = std::find(fTargets.begin(), fTargets.end(), target);
      |                                                                 ^
In file included from /usr/include/c++/10/iterator:66,
                 from /usr/include/string_theory/st_charbuffer.h:29,
                 from /usr/include/string_theory/st_string.h:26,
                 from /usr/include/string_theory/string:1,
                 from uru/libhsplasma/core/Stream/hsStream.h:23,
                 from uru/libhsplasma/core/Stream/pfPrcHelper.h:20,
                 from uru/libhsplasma/core/PRP/plCreatable.h:20,
                 from uru/libhsplasma/core/PRP/KeyedObject/hsKeyedObject.h:20,
                 from uru/libhsplasma/core/PRP/Object/plSynchedObject.h:20,
                 from uru/libhsplasma/core/PRP/Modifier/plModifier.h:20,
                 from uru/libhsplasma/core/PRP/Animation/plWindBoneMod.h:20,
                 from uru/libhsplasma/core/PRP/Animation/plWindBoneMod.cpp:17:
/usr/include/c++/10/bits/streambuf_iterator.h:422:5: note: candidate: ‘template<class _CharT2> typename __gnu_cxx::__enable_if<std::__is_char<_CharT2>::__value, std::istreambuf_iterator<_CharT> >::__type std::find(std::istreambuf_iterator<_CharT>, std::istreambuf_iterator<_CharT>, const _CharT2&)’
  422 |     find(istreambuf_iterator<_CharT> __first,
      |     ^~~~
/usr/include/c++/10/bits/streambuf_iterator.h:422:5: note:   template argument deduction/substitution failed:
uru/libhsplasma/core/PRP/Animation/plWindBoneMod.cpp:107:65: note:   ‘__gnu_cxx::__normal_iterator<plKey*, std::vector<plKey> >’ is not derived from ‘std::istreambuf_iterator<_CharT>’
  107 |     auto it = std::find(fTargets.begin(), fTargets.end(), target);
      |                                                                 ^
make[2]: *** [core/CMakeFiles/HSPlasma.dir/build.make:368: core/CMakeFiles/HSPlasma.dir/PRP/Animation/plWindBoneMod.cpp.o] Error 1
```
Version information:
```
$ g++ --version
g++ (Debian 10.2.0-15) 10.2.0
```

--- a/core/PRP/Animation/plWindBoneMod.cpp
+++ b/core/PRP/Animation/plWindBoneMod.cpp
@@ -15,6 +15,7 @@
  */
 
 #include "plWindBoneMod.h"
+#include <algorithm>
 
 /* plWindBoneOscillator */
 void plWindBoneOscillator::read(hsStream* S)

--- a/core/PRP/Animation/plWindBoneMod.cpp
+++ b/core/PRP/Animation/plWindBoneMod.cpp
@@ -20,30 +20,30 @@
 /* plWindBoneOscillator */
 void plWindBoneOscillator::read(hsStream* S)
 {
-    fUnknown1 = S->readFloat();
-    fUnknown2 = S->readFloat();
-    fUnknown3 = S->readFloat();
-    fUnknown4 = S->readFloat();
-    fUnknown5 = S->readFloat();
+    fDistX = S->readFloat();
+    fDistY = S->readFloat();
+    fDirX = S->readFloat();
+    fDirY = S->readFloat();
+    fSpeed = S->readFloat();
 }
 
 void plWindBoneOscillator::write(hsStream* S)
 {
-    S->writeFloat(fUnknown1);
-    S->writeFloat(fUnknown2);
-    S->writeFloat(fUnknown3);
-    S->writeFloat(fUnknown4);
-    S->writeFloat(fUnknown5);
+    S->writeFloat(fDistX);
+    S->writeFloat(fDistY);
+    S->writeFloat(fDirX);
+    S->writeFloat(fDirY);
+    S->writeFloat(fSpeed);
 }
 
 void plWindBoneOscillator::prcWrite(pfPrcHelper* prc)
 {
     prc->startTag("plWindBoneOscillator");
-    prc->writeParam("Unknown1", fUnknown1);
-    prc->writeParam("Unknown2", fUnknown2);
-    prc->writeParam("Unknown3", fUnknown3);
-    prc->writeParam("Unknown4", fUnknown4);
-    prc->writeParam("Unknown5", fUnknown5);
+    prc->writeParam("DistX", fDistX);
+    prc->writeParam("DistY", fDistY);
+    prc->writeParam("DirX", fDirX);
+    prc->writeParam("DirY", fDirY);
+    prc->writeParam("Speed", fSpeed);
     prc->endTag(true);
 }
 
@@ -52,11 +52,11 @@ void plWindBoneOscillator::prcParse(const pfPrcTag* tag)
     if (tag->getName() != "plWindBoneOscillator")
         throw pfPrcTagException(__FILE__, __LINE__, tag->getName());
 
-    fUnknown1 = tag->getParam("Unknown1", "0").to_float();
-    fUnknown2 = tag->getParam("Unknown2", "0").to_float();
-    fUnknown3 = tag->getParam("Unknown3", "0").to_float();
-    fUnknown4 = tag->getParam("Unknown4", "0").to_float();
-    fUnknown5 = tag->getParam("Unknown5", "0").to_float();
+    fDistX = tag->getParam("DistX", "0").to_float();
+    fDistY = tag->getParam("DistY", "0").to_float();
+    fDirX = tag->getParam("DirX", "0").to_float();
+    fDirY = tag->getParam("DirY", "0").to_float();
+    fSpeed = tag->getParam("Speed", "0").to_float();
 }
 
 

--- a/core/PRP/Animation/plWindBoneMod.h
+++ b/core/PRP/Animation/plWindBoneMod.h
@@ -22,11 +22,11 @@
 class PLASMA_DLL plWindBoneOscillator
 {
 protected:
-    float fUnknown1, fUnknown2, fUnknown3, fUnknown4, fUnknown5;
+    float fDistX, fDistY, fDirX, fDirY, fSpeed;
 
 public:
     plWindBoneOscillator()
-        : fUnknown1(), fUnknown2(), fUnknown3(), fUnknown4(), fUnknown5() { }
+        : fDistX(), fDistY(), fDirX(), fDirY(), fSpeed() { }
 
     void read(hsStream* S);
     void write(hsStream* S);

